src/pages/Onboard/NameTeam.vue

```diff
@@ -255,6 +344,83 @@ export default {
             </v-text-field>
           </v-col>
 
+          <v-col v-if="revealPendingTeams" key="pendingInvites" cols="12">
+            <div v-if="pendingInvitations.length" class="body-1">
+              Your pending invitations:
+            </div>
+            <v-list
+              v-for="pt in pendingInvitations"
+              :key="pt.id"
+              color="transparent"
+            >
+              <v-list-item
+                style="align-items: baseline;
+                color: #fff !important;
+                justify-content: space-between;"
+              >
+                <span>{{ pt.tenant.name }}</span>
+                <div class="mt-8">
+                  <v-btn
+                    class="mr-3"
+                    color="accentPink"
+                    dark
+                    depressed
+                    :loading="accepting"
+                    @click="accept(pt)"
```

Comment:
If therre's more than one invitation, all the buttons are set to loading here.  Could we have a loading id that's specific to each button/pending tenant?

Reply:
should be fixed